src/dimension.c

```diff
@@ -1337,6 +1380,7 @@ static ScanTupleResult
 dimension_rename_schema_name(TupleInfo *ti, void *data)
 {
 	HeapTuple tuple = heap_copytuple(ti->tuple);
+	// TODO:: Q:: now Dimension table may have null values so this may not be valid??
 	FormData_dimension *dimension = (FormData_dimension *) GETSTRUCT(tuple);
```

Comment:
now Dimension table may have null values so is this still valid or should I change it to `heap_getattr` or `heap_deform_tuple` ?